tmd/io/tests/test_io.py

```diff
@@ -85,4 +87,11 @@ def test_io_load():
 def test_load_population():
     population = io.load_population(POP_PATH)
     nt.ok_(len(population.neurons) == 5)
-    names = np.array([n.name for n in population.neurons])
+    names = np.array([n.name.split('/')[-1] for n in population.neurons])
+    
+    L = glob.glob(POP_PATH + '/*')
+    population1 = io.load_population(L)
+    nt.ok_(len(population.neurons) == 5)
+    names1 = np.array([n.name.split('/')[-1] for n in population1.neurons])
```

Comment:
you could use os.path.basename no ?

Reply:
I think it should be the same. I can modify accordingly

Reply:
I think so but the intent is clearer when reading the code.